tools/shared.py

```diff
@@ -2267,16 +2267,18 @@ def emar(action, output_filename, filenames, stdout=None, stderr=None, env=None)
       assert os.path.exists(output_filename), 'emar could not create output file: ' + output_filename
 
   @staticmethod
-  def emscripten(filename, append_ext=True, extra_args=[]):
+  def emscripten(filename, js_libraries):
     if path_from_root() not in sys.path:
       sys.path += [path_from_root()]
     import emscripten
     # Run Emscripten
-    cmdline = [filename + ('.o.ll' if append_ext else ''), '-o', filename + '.o.js'] + extra_args
+    infile = filename
+    outfile = filename + '.o.js'
     if jsrun.TRACK_PROCESS_SPAWNS:
-      logging.info('Executing emscripten.py compiler with cmdline "' + ' '.join(cmdline) + '"')
+      logging.info('Executing emscripten.py compiler with cmdline "' + ' '.join([infile, outfile]) + '"')
```

Comment:
Does this log even make sense now? It's not tracking a process spawn, and it's being called with args, not a cmdline.